mlkem/native/aarch64/ntt_clean.S

```diff
@@ -204,6 +214,20 @@ _ntt_asm_clean:
         load_roots_123
 
         .p2align 2
+
+        // Bounds reasoning:
+        // - There are 7 layers
+        // - When passing from layer N to layer N+1, each layer-N value
+        //   is modified through the addition/subtraction of a Montgomery
+        //   product of a twiddle of absolute value < q/2 and a layer-N value.
+        // - Recalling that |fqmul(a,t)| < q * (0.0254*C + 1/2) for
```

Comment:
You lost me at this point. What is C? You're using "^" to mean "power" not "exclusive or", right? That's confusing for C programmers.

Reply:
Yes, meant it as "power". Maybe "**" will be more universally understood?

Reply:
Possibly, but ** gets mangled because it means something special in MarkDown format.

Reply:
Ok, I'll leave it as is for now.